src/main.rs

```diff
@@ -40,10 +40,10 @@ const MENU: [&'static str; DISP_HEIGHT as usize] = [
     "║ - a/d - left/right ║",
     "║   - q/e - rotate   ║",
     "║ - s -> drop piece  ║",
+    "║    - p -> pause    ║",
```

Comment:
Nothing wrong with this, but I wanted to note that the "p for pause" thing should be added to the README as well under the `Controls` header